tmt/steps/execute/internal.py

```diff
@@ -61,6 +62,132 @@
 """
 FILE_SUBMIT_NAME = "tmt-file-submit"
 
+RSTRNT_REPORT_RESULT_SCRIPT_PATHS = (
```

Comment:
Just a suggestion: lets get rid of the `/usr/local/bin` inside all of these paths + reboot paths and prefix it automatically in the code setting it up. All of the compatibility layer will be in the same directory anyway.

Reply:
That makes sense. I've made that change.